src/io/flutter/run/daemon/DaemonApi.java

```diff
@@ -135,12 +145,12 @@ else if (outputType.equals(ProcessOutputTypes.STDOUT)) {
       }
 
       @Override
-      public void processWillTerminate(ProcessEvent event, boolean willBeDestroyed) {
+      public void processWillTerminate(@NotNull ProcessEvent event, boolean willBeDestroyed) {
```

Comment:
Ah.  This is good.  I've been meaning to look and see if this is annotated in `2017.3`.  (I think we want that to be true?)

Reply:
EDIT: I took a quick look and think we're good. 👍

Reply:
awesome, thanks for checking. I remember a bunch of annotations being introduced in 2017.3, so somewhat assume that missing annotations are not safe to start adding in...